processing/src/main/java/io/druid/segment/serde/ComplexColumnPartSerde.java

```
@@ -58,9 +68,25 @@ public String getTypeName()
     return typeName;
   }
 
-  public static SerializerBuilder serializerBuilder()
+  @Override
+  public Serializer getSerializer()
   {
-    return new SerializerBuilder();
+    return serializer;
+  }
+
+  @Override
+  public Deserializer getDeserializer()
+  {
+    return new Deserializer()
+    {
+      @Override
+      public void read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnConfig)
+      {
+        if (serde != null) {
```

Why silently ignore if `serde` is null?